core-js/src/main/javascript/cdf/components/ToggleButtonBaseComponent.js

```diff
@@ -90,7 +90,7 @@ define(['../lib/jquery', './InputBaseComponent'], function($, InputBaseComponent
         input.appendTo(li);
         li.append($("<label/>")
           .attr({'for': myself.name + i})
-          .text(myArray[i][1]));
+          .html(myArray[i][1]));
```

Comment:
For this case we receive escaped label, for example: Anna&amp;#39;s Decorations, Ltd
For correct presentation need use html representation instead of text (text does not convert this text to apostrophe).
For example, SelectBaseComponent also uses html method.